.gitignore

```diff
@@ -0,0 +1,4 @@
+*.pdf
+hits*.csv
+*.txt
+build/
```

Comment:
I think this is fine, but please note that the build dir does contain one file

Reply:
Tracking of build/initial is back on